traffic_portal/test/integration/PageObjects/ServersPage.po.ts

```diff
@@ -141,8 +141,7 @@ export class ServersPage extends BasePage {
     let name = nameServer+this.randomize;
     await this.txtQuickSearch.clear();
     await this.txtQuickSearch.sendKeys(name);
-    await browser.actions().mouseMove(element(by.xpath("//span[text()='"+name+"']"))).perform();
-    await browser.actions().doubleClick(element(by.xpath("//span[text()='"+name+"']"))).perform();
+    await browser.actions().click(element(by.xpath("//span[text()='"+name+"']"))).perform();
```

Comment:
We shouldn't be using XPath, CSS is much more familiar and easier to understand. The equivalent selector in this case would be `by.cssContainingText("span", name)`, which is much simpler.